### ISSUE TITLE
Noise in IMU orientation 

### ISSUE THREAD

**Issue by SijmenHuizenga** [FS-Online/Formula-Student-Driverless-Simulator#143]
[I'm a bit surprised to see there is no noise in the orientation of the imu.](https://github.com/FS-Online/Formula-Student-Driverless-Simulator/blob/master/AirSim/AirLib/include/sensors/imu/ImuSimple.hpp#L62-L64). [This is in line with the docs](https://fs-online.github.io/Formula-Student-Driverless-Simulator/latest/imu/) . @skimslozo @davidoort do you remember if this is correct?

**Comment by davidoort** [comment]
This is correct. At the moment there is no noise in the orientation. If teams want an extra challenge adding noise here could be a way to increase the difficulty of the competition

**Comment by SijmenHuizenga** [comment]
Oke! Good to know!
